SwiftWisdom/Core/UIKit/UIView+Geometry.swift

```diff
@@ -0,0 +1,229 @@
+//
+//  UIView+Geometry.swift
+//  SwiftWisdom
+//
+//  Created by Logan Wright on 1/19/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import UIKit
+
+extension UIView {
+    public var ip_width: CGFloat {
+        return CGRectGetWidth(frame)
```

Comment:
for width, bounds and height should always be equal.  For whatever reason though, bounds feels more right here, so I'll swap it out.